tests/test_systemd.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
```

Comment:
Please squash this "add systemd test" commit into the previous one that adds the template.

Reply:
Squashed.